services/frontend-service/src/assets/_components.scss

```diff
@@ -41,5 +41,6 @@ Copyright 2023 freiheit.com*/
 @import 'src/ui/components/Spinner/Spinner';
 @import 'src/ui/components/EnvironmentLockDisplay/EnvironmentLockDisplay';
 @import 'src/ui/components/ReleaseVersion/ReleaseVersion';
```

Comment:
please include a picture in the PR description of the new look (and if it makes sense also one screenshot for how it looked before)